test_suite/tests/test_borrow.rs

```diff
@@ -4,8 +4,11 @@
     clippy::used_underscore_binding
 )]
 
-use serde::de::value::{BorrowedStrDeserializer, MapDeserializer};
-use serde::de::{Deserialize, Deserializer, IntoDeserializer};
+use serde::de::value::{
+    BorrowedBytesDeserializer, BorrowedStrDeserializer, CowBytesVisitor, CowStrVisitor, Error,
+    MapDeserializer,
+};
+use serde::de::{Deserialize, DeserializeSeed, Deserializer, IntoDeserializer};
```

Comment:
`DeserializeSeed` is required to be in scope to be able to call it methods on `Cow...Visitor`s